### DISCUSSION TITLE
How could I customize the cursor when hovering and dragging

### DISCUSSION THREAD

**Discussion by Trav1sRen** [clauderic/dnd-kit#1175] [Q&A]
Currently the cursor is `default` when hovering or dragging the item.
Is there anyway to make the cursor `grab` on hovering and `grabbing` on dragging?
Like the behavior demonstrated in the Stroybook 
https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-grid--basic-setup

I tried using the `isDragging` state deconstructed from the return value of the `useSortable` hooks. And change the inline style of `cursor` according to the `isDragging`, but only the hover cursor works. The cursor gets back to `default` when dragging the item.

**Comment by nuthinking** [comment]
On Chrome the example you linked works already as you would like it to work.
